content/en/security/agentless_scanning/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Cloud Security Management Agentless Scanning
+kind: documentation
+further_reading:
+- link: "/security/cloud_security_management/setup/agentless_scanning"
+  tag: "Documentation"
+  text: "Setting up Agentless Scanning"
+- link: "TBD"
+  tag: "Blog"
+  text: "Detect vulnerabilities in minutes with Datadog Agentless scanning for CSM"
+- link: "/security/vulnerabilities"
+  tag: "Documentation"
+  text: "Read more about CSM Vulnerabilities"
+---
+
+<div class="alert alert-info">CSM Agentless Scanning is in beta.</div>
```

Comment:
"Agentless Scanning for Cloud Security Management is in public beta for AWS cloud environments."

Want to emphasize that it is for AWS only.

Reply:
Done!